### DISCUSSION TITLE
How to apply new line styling to a piece of content which is updated from js file ?

### DISCUSSION THREAD

**Discussion by binilvjacob** [fastn-stack/fastn#1338] [Q&A]
How can I update content that has several lines from js ? When I use the window.ftd.set_value() method new lines are not coming up. Instead of directly giving this content I am using a string variable and later on from the js file, I am updating the value of this string variable. So then the content gets changed, but the new lines are missing. Every lines are shown as a single para.

**Comment by amitu** [comment]
This would be tricky in 0.3. 0.4 should be possible, not sure if it works as it is not tested in 0.4 yet. @binilvjacob can you create some issues you encounter when you test 0.3 to 0.4. We want everyone to migrate to 0.4, supporting 0.3 indefinitely would be hard. Next release, 0.4 -> 0.5 we are planning to add edition support so both will be supported, but 0.3 is hard to support for long term.
